### DISCUSSION TITLE
Question about correlation between Gene Activity in scATACseq and RNA expression

### DISCUSSION THREAD

**Discussion by HugoGonzalezVelozo** [stuart-lab/signac#1333] [Q&A]
Hello, 
I recently run some human samples using scATACseq and analyzed them using Signac. It happens that I also have scRNAseq data from the same samples (separated experiments). Now, I want to compare how gene activity inferred by Signac, correlates with RNA expression in the same target populations. 
Question:  Is it valid to use the function AverageExpression to calculate average gene activity per gene in Signac (see below)? 

GeneAct_avgExp <- AverageExpression(
  Object,
  assays = "RNA",
  features = NULL,
  return.seurat = FALSE,
  group.by = "ident",
  add.ident = NULL,
  slot = "data",
  verbose = TRUE)

I run the script above and it works, but I am not sure if it is right, then I did the same in the equivalent population on the scRNAseq data, and finally, I run a correlation among the two modalities (excluding genes missing across modalities). The problem is that by doing this, the correlation is incredibly low, close to 0.03. It looks like there is almost no correlation between inferred gene activity by Signac and the expression of the given gene by scRNAseq in the same sample. I am doing something wrong? 

Thanks in advance,

Hugo

**Comment by timoast** [comment]
Running `AverageExpression` should be ok. It's certainly expected that the correlation between gene activity and gene expression may not be very high. The motivation for the original development of the gene activity score was for use in multimodal integration, where information is share across all the genes to identify similar cell states across datasets.

I'm also not sure how you're running the correlation here, are you testing the correlation across genes within a cell type for activity vs expression, or within a gene what is the correlation between expression and activity across cell types? I would expect higher values for the latter.

**Reply by HugoGonzalezVelozo** [reply]
Thanks for the reply.  I am testing the correlation of the expression (RNA and Gene Score) of all genes in the same population. It is just an exploratory analysis trying to understand the gene score. But, I was just surprised at how low the correlation was, I expected that genes with high expression would also have a high gene score and vice-versa with low expressed genes.  Ultimately, what I would like to do is to classify cells by cell cycle score using gene score, but I will open a new discussion about that topic.
